blackbox/docs/sql/reference/create_user.txt

```diff
@@ -35,3 +39,14 @@ Parameters
 :username: The unique name of the database user. The name follows the
            principles of a SQL identifier (see
            :ref:`sql_lexical_keywords_identifiers`).
+
+Clauses
+=======
+
+``WITH``
+--------
+
+The following ``user_parameter`` are supported to define a new user account:
+
+:password: The password as cleartext entered as string literal (surrounded by
+           single quotes ``'``).
```

Comment:
do we have to state here that string literals must be surrounded by single quotes? think it can lead to confusions  as the definition of string literals already contain single quotes.